Dockerfile

```diff
@@ -5,7 +5,7 @@
 # image with tools only needed during the image build.
 
 # First build the temporary image.
-FROM python:3.7-slim-buster AS builder
+FROM python:3.10.8-slim-buster AS builder
```

Comment:
It'd be safe to make this just `3.10` since we didn't pin patch releases previously.